include/operations/blas_constants.hpp

```diff
@@ -39,12 +39,35 @@
 #define SYCLBLAS_DEFINE_CONSTANT(type, indicator, val) \
   template <>                                          \
   struct constant<type, indicator> {                   \
-    static constexpr type value = val;                 \
-  };
+    static const type value;                           \
+  };                                                   \
+  const type constant<type, indicator>::value =        \
+      val;  // temporary work around for duplicator issue will be fixed int the
+            // next release
```

Comment:
"Temporary work around for a compiler issue on ComputeCpp 0.2.0, bug reported internally"